tests/functional/u/unnecessary/unnecessary_ellipsis.txt

```diff
@@ -0,0 +1,3 @@
+unnecessary-ellipsis:9:4:9:7::Unnecessary ellipsis constant:UNDEFINED
+unnecessary-ellipsis:43:4:43:7:docstring_and_ellipsis:Unnecessary ellipsis constant:UNDEFINED
+unnecessary-ellipsis:59:4:59:7:DocstringAndEllipsis:Unnecessary ellipsis constant:UNDEFINED
```

Comment:
@Pierre-Sassoulas This makes me start to wonder if we can add confidence to this message? Since it isn't based on inference?

Reply:
Shouldn't we first add a new ``NO_INFERENCE`` value ?

Reply:
Probably! Let's leave this for now.